tools/benchmark/src/MochaReporter.ts

```diff
@@ -42,8 +42,8 @@ function getName(name: string): string {
  */
 // eslint-disable-next-line @typescript-eslint/no-extraneous-class
 module.exports = class {
-    public constructor(runner: Runner, options?: ReporterOptions) {
-        const benchmarkReporter = new BenchmarkReporter(options?.reportDir);
+    public constructor(runner: Runner, options?: { [key: string]: any; }) {
```

Comment:
In case people want to use this programmatically, keeping `ReporterOptions` around in `Configuration.ts` and typing the options object as such instead seems better:

```suggestion
    public constructor(runner: Runner, options?: { reporterOptions?: ReporterOptions }) {
```